pkg/network/http/etw_interface.go

```diff
@@ -0,0 +1,124 @@
+// Unless explicitly stated otherwise all files in this repository are licensed
+// under the Apache License Version 2.0.
+// This product includes software developed at Datadog (https://www.datadoghq.com/).
+// Copyright 2016-present Datadog, Inc.
+
+//go:build windows && npm
+// +build windows,npm
+
+package http
+
+import (
+	"sync"
+	"time"
+
+	"github.com/DataDog/datadog-agent/pkg/network/config"
+	"github.com/DataDog/datadog-agent/pkg/network/etw"
+	"github.com/DataDog/datadog-agent/pkg/util/log"
+)
+
+type httpEtwInterface struct {
+	maxEntriesBuffered int
+	dataChannel        chan []etw.Http
+	eventLoopWG        sync.WaitGroup
+}
+
+func newHttpEtwInterface(c *config.Config) *httpEtwInterface {
+	return &httpEtwInterface{
+		maxEntriesBuffered: c.MaxHTTPStatsBuffered,
+		dataChannel:        make(chan []etw.Http),
```

Comment:
I think this suffers from the same race condition as the driver interface - we should consider removing this channel & processing synchronously here as well

Reply:
@ISauve For now, I'm going to leave this like this.
I have a follow-on PR that does some better converging of the linux/windows.
And then I want to re-examine how we do this, and whether the channel is necessary.
If nothing else, it will need some different locking, but I don't want to make this PR more crazy than it already is.

Reply:
Sounds good to me!